plugins/kubernetes/src/kubernetes-auth-provider/GoogleKubernetesAuthProvider.ts

```diff
@@ -39,7 +39,7 @@ export class GoogleKubernetesAuthProvider implements KubernetesAuthProvider {
   async getCredentials(): Promise<{ token: string }> {
     return {
       token: await this.authProvider.getAccessToken(
-        'https://www.googleapis.com/auth/cloud-platform',
```

Comment:
I wonder would this prevent us from doing patches/deletes against objects through the proxy API when using this auth @jamieklassen

Reply:
lol of course it would yep

Reply:
Should we have `read-only` auth scope value as the default?
 Will the configuration should look like this one?:
 ```yaml
  clusterLocatorMethods:
    - type: 'gke'
      projectId: 'my-project'
      authProvider: google
      authScopes: []
 ```

Reply:
Actually Google's authN model might not work the way I thought. When I checked out this code, visited the Kubernetes tab, opened my dev tools and copied the access token used to get objects for an entity, I was able to use the same token to create a namespace.

Reply:
Sure we can merge, if we discover issues we can raise a new bug